test/runtests.jl

```diff
@@ -1,6 +1,6 @@
 using Test
 using BSeries
-
+include("../src/Energy_Preserving.jl")
```

Comment:
No. The test code must not `include` code from `src` manually. This has to be handled from within the package